pkg/scheduler/scheduler_test.go

```diff
@@ -185,71 +185,104 @@ func TestScheduler(t *testing.T) {
 	}
 
 	for i, item := range table {
-		var gotError error
-		var gotPod *v1.Pod
-		var gotForgetPod *v1.Pod
-		var gotAssumedPod *v1.Pod
-		var gotBinding *v1.Binding
-		configurator := &FakeConfigurator{
-			Config: &Config{
-				SchedulerCache: &schedulertesting.FakeCache{
-					ForgetFunc: func(pod *v1.Pod) {
-						gotForgetPod = pod
-					},
-					AssumeFunc: func(pod *v1.Pod) {
-						gotAssumedPod = pod
-					},
-				},
-				NodeLister: schedulertesting.FakeNodeLister(
-					[]*v1.Node{&testNode},
-				),
-				Algorithm: item.algo,
-				GetBinder: func(pod *v1.Pod) Binder {
-					return fakeBinder{func(b *v1.Binding) error {
-						gotBinding = b
-						return item.injectBindError
-					}}
-				},
-				PodConditionUpdater: fakePodConditionUpdater{},
-				Error: func(p *v1.Pod, err error) {
-					gotPod = p
-					gotError = err
+		name := fmt.Sprintf("%v:expecterror=%v:injecterror=%v:eventreason=%v", i, item.expectError, item.injectBindError, item.eventReason)
```

Comment:
Please give a meaningful but short name here, the current name is too long.

Reply:
👍